script.module.tmdbsimple/addon.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.tmdbsimple"
+       name="tmdbsimple"
+       version="2.2.0"
+       provider-name="Celia Oakley, rmrector">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+    <import addon="script.module.requests" version="2.15.1"/>
+  </requires>
+  <extension point="xbmc.python.module"
+             library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">A wrapper for The Movie Database API v3</summary>
+    <description lang="en">Packed for Kodi from https://github.com/celiao/tmdbsimple</description>
+    <platform>all</platform>
+    <language></language>
+    <license>GPL-3.0</license>
+    <forum></forum>
+    <website>https://github.com/celiao/tmdbsimple</website>
```

Comment:
This might be confusing about the real upstream location. Maybe point both to your repo?